src/concrete/ml/quantization/quantized_ops.py

```diff
@@ -1764,73 +1782,6 @@ def q_impl(
 
         return sum_qarray
 
-    def _prepare_inputs_with_constants(
```

Comment:
why all this has gone ?

Reply:
I mostly summarized it in the PR's description, but we've discussed about it with @jfrery and @fd0r following https://github.com/zama-ai/concrete-ml-internal/issues/4094 (if you want more info : https://zama-ai.slack.com/archives/C0455QDGT6C/p1702542206104809)

Reply:
thanks